utils/node_local_test.py

```diff
@@ -3436,8 +3436,8 @@ def run(wf, args):
         server = DaosServer(conf, test_class='no-debug')
         server.start()
         if fi_test:
-#            fatal_errors.add_result(test_alloc_fail_copy(server, conf,
-#                                                         wf_client))
+            #fatal_errors.add_result(test_alloc_fail_copy(server, conf,
+            #                                             wf_client))
```

Comment:
Whitespace only change.

Reply:
@ashleypittman Since there are minor I can address these in a follow on PR I have for this file. Some of the parsing for all the datamover tools broke after PR #6333 was merged and there is a fix in here for it.

Reply:
I only mention it because it's going to conflict with one of mine.